frontend/src/routes/Governance/discovered/ByCluster/common.tsx

```diff
@@ -0,0 +1,201 @@
+          // Parent Policy exist
+          if (labelVal?.split('.').length == 2) {
+            return (
+              <Link
+                to={generatePath(NavigationPath.policyTemplateDetails, {
+                  namespace: labelVal.split('.')[0],
+                  // `kind: Policy` name
+                  name: labelVal.split('.')[1],
+                  clusterName: item.cluster,
+                  apiVersion: item.apiversion,
+                  apiGroup: item.apigroup,
+                  kind: item.kind,
+                  // discovered policy name
+                  templateName: item.name,
+                })}
+              >
+                {item.cluster}
+              </Link>
+            )
+          }
+          return (
+            <Link
+              to={generatePath(NavigationPath.discoveredPolicyDetails, {
+                clusterName: item.cluster,
+                apiVersion: item.apiversion,
+                apiGroup: item.apigroup,
+                kind: item.kind,
+                // discovered policy name
+                templateName: item.name,
+              })}
+            >
+              {item.cluster}
+            </Link>
+          )
+        },
+        sort: 'cluster',
+        search: 'cluster',
+        id: 'cluster',
+        exportContent: (item) => item.cluster,
+      },
+      ...(moreCols ?? []),
+      {
+        header: t('Response action'),
+        cell: 'remediationAction',
+        sort: 'remediationAction',
+        search: 'remediationAction',
+        id: 'responseAction',
+        exportContent: (item: DiscoverdPolicyItem) => item.remediationAction,
+      },
+      {
+        header: t('Violations'),
+        cell: (item: DiscoverdPolicyItem) => {
+          const compliant = item?.compliant?.toLocaleLowerCase()
+          switch (compliant) {
+            case 'compliant':
+              return (
+                <div>
+                  <CheckCircleIcon color="var(--pf-global--success-color--100)" /> {t('No violations')}
+                </div>
+              )
+            case 'noncompliant':
+              return (
+                <div>
+                  <ExclamationCircleIcon color="var(--pf-global--danger-color--100)" /> {t('Violations')}
+                </div>
+              )
+            case 'pending':
+              return (
+                <div>
+                  <ExclamationTriangleIcon color="var(--pf-global--warning-color--100)" /> {t('Pending')}
+                </div>
+              )
+            default:
+              return (
+                <div>
+                  <ExclamationTriangleIcon color="var(--pf-global--warning-color--100)" /> {t('No status')}
+                </div>
+              )
+          }
+        },
+        sort: 'compliant',
+        id: 'violations',
+        exportContent: (item: DiscoverdPolicyItem) => item.compliant || '-',
+      },
+      {
+        header: t('Severity'),
+        // TODO Add severity icon
```

Comment:
Still todo? (I'm fine leaving the TODO if it needs to wait for something, I'm just wondering)

Reply:
Console team is preparing for this svgs